app/controllers/submissions_controller.rb

```diff
@@ -611,6 +612,28 @@ def view
     end
   end
 
+  action_auth_level :releaseSubmission, :course_assistant
+  def releaseSubmission
```

Comment:
Maybe call this `release_student_grade` to be more descriptive, also use snake case to be more consistent (I know there are other endpoints with camel casing but I think snake case is better)